Data Section Update (#1127)
ACCESS-Hive/access-hive.github.io: docs/model_evaluation/data/finding/access_nri_intake.md

# What is the ACCESS-NRI Intake Catalogue?

The ACCESS-NRI Intake Catalogue is essentially a Python interface to the table of
climate data products that form the ACCESS Data Catalogue on _Gadi_.
For detailed information, Python tutorials and more, please see the
ACCESS-NRI Intake Catalogue [documentation](https://access-nri-intake-catalog.readthedocs.io/en/latest/index.html).

> Each entry in the table corresponds to a different product, where the columns contain attributes associated with that product (e.g. available models, frequencies and data variables).

Reviewer: ```suggestion
Each catalogue entry corresponds to a different data product, where the columns describe the attributes associated with that product (e.g., available models, frequencies and data variables).
```

> Users can search these attributes to find the products that might be useful to them.

Reviewer: ```suggestion
Users can filter or search these attributes to find the products relevant to their needs.
```

> For example, a user may want to know which data products contain variables X, Y and Z at monthly frequency.

Reviewer: ```suggestion
For example, a user could search for a data products containing variables `X`, `Y` and `Z` at a `monthly` frequency.
```

The ACCESS-NRI Intake Catalogue enables users to find products that satisfy their
needs and to subsequently load the data with Python without having to know the

> location or structure of the underlying files.

Reviewer: ```suggestion
The ACCESS-NRI Intake Catalogue also enables users to directly load the desired data in Python, without having to know the location or structure of the underlying files.
```
